stephanie-smith/short-response.md

```diff
@@ -0,0 +1,66 @@
+# Problem Set 2.2 - Variables and Control Flow
+## Short Response Questions
+
+1. **What does the code below log? Why?**
+   ```javascript
+    {
+      let singer = 'Omar Apollo';
+    }
+
+    console.log(`My favorite singer is ${singer}`);
+   ```
+   - It logs "My favorite singer is Omar Apollo".
+
+2. **What happens when we run the following code? Why?**
+   ```javascript
+   const favorite = 'Juan Pablo';
+   console.log(`Our favorite Marcy Lab family member is ${favorite}!`);
+
+   favorite = 'Maya';
+   console.log(`Actually, ${favorite} is my favorite.`);
+   ```
+   - When you run the code, you get an error about the type used for the variable `favorite`, which is a constant. This means they cannot be redefined.
+
+3. **What does the following code log? Why?**
+   ```javascript
+   let score = 90; 
+   let newScore = score;
+   score = 100; 
+   console.log(score, newScore);
+   ```
+   - It logs 100 and 90. The variable `score` starts off as 90, while a new variable `newScore` is defined as `score`. This means both `score` and `newScore` equal 90. Then, `score` is redefined as 100. The value in `newScore` does not change to 100 despite the newly assigned value, because of the way the code cascades.
+
+4. **Why doesn't the following code log an uppercase string? Change the code so that it does.**
+   ```javascript
+   let greet = 'hello';
+   let scream = greet.toUpperCase();
+   console.log(scream); 
+   ```
+   - The function must be called when defining the variable `scream`, otherwise it won't work.
```

Comment:
I see what you're saying here. Good job. Let's make this a bit more clear.
>The `greet.toUpperCase()` method returns `HELLO` but that value is not stored anywhere. To solve this problem I assign the returned value of this method call to a variable `scream`.

Use my example above to edit your answer.